abandoned_checkout.go

```diff
@@ -47,10 +47,10 @@ type AbandonedCheckout struct {
 	Currency                 string               `json:"currency,omitempty"`
 	CompletedAt              *time.Time           `json:"completed_at,omitempty"`
 	ClosedAt                 *time.Time           `json:"closed_at,omitempty"`
-	UserID                   int64                `json:"user_id,omitempty"`
+	UserId                   uint64               `json:"user_id,omitempty"`
```

Comment:
Apologies if it's readily apparent elsewhere, but what prompted the change to uint64?

Reply:
https://github.com/bold-commerce/go-shopify/issues/106

Reply:
Changed all IDs to uint64